test_common/helpers/setup.rb

```diff
@@ -159,6 +159,44 @@ def stub_et_api # rubocop:disable Metrics/MethodLength
           )
       end
 
+      def stub_valid_office_code
+        stub_request(:post, "#{ENV.fetch('ET_API_URL', 'http://api.et.127.0.0.1.nip.io:3100/api')}/v2/validate_response").
+          with(headers: { 'Content-Type': 'application/json', 'Accept': 'application/json' }).
+          to_return(
+            headers: { 'Content-Type': 'application/json' },
+            body:
+              {
+                "status": "accepted",
+                "errors": []
+              }.to_json
+          )
+      end
+
+      # Stub submission call to API with invalid office code
+      def stub_invalid_office_code
+        stub_request(:post, "#{ENV.fetch('ET_API_URL', 'http://api.et.127.0.0.1.nip.io:3100/api')}/v2/validate_response").
+          with(headers: {'Content-Type': 'application/json', 'Accept': 'application/json'},
+               body: {
+                 "case_number": "0254321/2018"
+               }).
+          to_return(
+            headers: {'Content-Type': 'application/json'},
+            body:
+              {
+                "status": "not_accepted",
+                "errors": [
+                  {
+                    "status": 422,
+                    "code": "invalid_office_code",
+                    "title": "Invalid case number",
+                    "detail": "Invalid case number - invalid office code",
+                    "source": "/data/case_number"
```

Comment:
You are stating that the error exists in data/case_number - but it was provided in just case_number ? (i.e. no 'data')